assemble/bin/accumulo

```diff
@@ -58,6 +58,18 @@ function main() {
     echo "$CLASSPATH"
     exit 0
   fi
+  
+  # Set up path variable for default import config file
+  export jShellPath="$conf/jshell-init.jsh"
+  
+  if [[ $cmd == "jshell" ]]; then
+    shift
+    if [[ -f "$jShellPath" ]]; then
+      exec "$cmd" --startup DEFAULT "$jShellPath" "$@"
```

Comment:
I just learned that we can actually specify more than one `--startup` option, so if we don't want to customize the `java.*` imports for now, and just want the defaults, we can do:

```suggestion
      exec "$cmd" --startup DEFAULT --startup "$jShellPath" "$@"
```

Reply:
Yeah for the DEFAULT script, I originally left that as a start up parameter to ensure the user can see the default import library and the Accumulo API when they first boot up JShell. I believe having two start-up statements should do the trick. Also I believe I found a work around regarding the Scanner. Whenever a Scanner is defined, we need to explicitly specify which Scanner are we referring to. For example: 

**Referencing Accumulo Scanner:**
(org.apache.accumulo.core.client.Scanner scan = client.createScanner("GothamPD", Authorizations.EMPTY);

**Referencing Java Scanner:**
 java.util.Scanner myObj = new java.util.Scanner(System.in);

I created a Java test code where there one function uses the Accumulo Scanner and another function uses the Java Scanner. I then executed the code in JShell with Accumulo and both functions where able to execute with no issues.

Reply:
I was aware of the workaround, but it's still annoying to specify the fully-qualified class name. Since the java.util.Scanner is very unlikely to be used by Accumulo devs using this, I was thinking we could try to exclude it by replacing `java.util.*` from the defaults with a narrower set of useful imports from `java.util`, but I'm not sure what that set would look like. It would probably include a bunch of Collection classes, but I'm not sure what else.

The alternative is that we just settle for the workaround for now, and consider a new name for Scanner in Accumulo's public API in the future to make it easier to coexist with java.util.Scanner.

Reply:
Would you like me to manually import all the classes from java.util.* into the jshell-init.jsh file or just go with the alternative solution?

Reply:
Let's just go with the double-startup option for now. `--startup DEFAULT --startup "$jShellPath"`

Reply:
Cool. I just pushed a commit to apply your double-startup option idea into the accumulo script file.